packages/gatsby/src/state-machines/query-running/index.ts

```diff
@@ -57,7 +60,7 @@ export const queryStates: MachineConfig<IQueryRunningContext, any, any> = {
         },
       },
     },
-
+    // This waits for the jobs API to finish
     waitingForJobs: {
```

Comment:
It put it here because I considered that as the jobs are kicked-off by the query running, the query running state shouldn't end until they're complete. Does that make sense?

Reply:
it make sense but that's not entirely true. It could be kicked of anywhere in code. It should end right before we go back to idle.

Reply:
Ah, ok. In that case I'll move it up to the top level, just before idle.